docs/current_docs/quickstart/agent/part2.mdx

```diff
@@ -0,0 +1,377 @@
+build --source $completed | as-service | up --ports 8080:80
+# Save the changes to your filesystem
+$completed | export .
+```
+
+Now you have built an agent that can write code for your project!
+
+## Deploy the agent to GitHub
+
+The next step is to take this agent and deploy it so that it can run automatically. For this part of the guide, you will configure Github so that when you label a Github issue with "develop", the agent will automatically pick up the task and open a pull request with its solution.
+
+The following steps require that you used the Github template and have your own hello-dagger repository.
+
+### Create the develop-issue function
+
+The `develop-issue` function will use a dependency from [Daggerverse](https://daggerverse.dev) called [github-issue](https://daggerverse.dev/mod/github.com/kpenfound/dag/github-issue@b316e472d3de5bb0e54fe3991be68dc85e33ef38). This module gives the functions needed to read Github issues and create pull requests.
+
+Install the dependency:
+
+```shell
+dagger install github.com/kpenfound/dag/github-issue@b316e472d3de5bb0e54fe3991be68dc85e33ef38
+```
+
+<Tabs groupId="language" queryString="sdk">
+    <TabItem value="go" label="Go">
+    Add the following new function to the main dagger module in `.dagger/main.go`:
+
+    ```go file=./snippets/part3/agent/go/agent.go.snippet
+    ```
+    </TabItem>
+    <TabItem value="python" label="Python">
+    Add the following new function to the main dagger module in `.dagger/src/hello_dagger/main.py`:
+
+    ```python file=./snippets/part3/agent/python/agent.py.snippet
+    ```
+    </TabItem>
+    <TabItem value="typescript" label="Typescript">
+    Add the following new function to the main dagger module in `.dagger/src/index.ts`:
+
+    ```typescript file=./snippets/part3/agent/typescript/agent.ts.snippet
+    ```
+    </TabItem>
+    <TabItem value="php" label="PHP">
+    Add the following new function to the main dagger module in `.dagger/src/HelloDagger.php`:
+
+    ```php file=./snippets/part3/agent/php/agent.php.snippet
+    ```
+
+    Be sure to add imports for the following:
+
+    ```php
+    use Dagger\Secret;
+    ```
+    </TabItem>
+    <TabItem value="java" label="Java">
+    Add the following new function to the main dagger module in `.dagger/src/main/java/io/dagger/modules/hellodagger/HelloDagger.java`:
+
+    ```java file=./snippets/part3/agent/java/agent.java.snippet
+    ```
+
+    Be sure to add imports for the following:
+
+    ```java
+import io.dagger.client.Secret;
+import io.dagger.client.GithubIssue;
+    ```
+    </TabItem>
+</Tabs>
+
+The `develop-issue` function connects the dots in the automation flow. It will be called when a Github issue is ready to be developed. Using the `github-issue` dependency, this code will:
+
+- Read the title and body from the Github issue
+- Use the body as the assignment for the agent
+- Open a pull request with the Directory returned by the agent
+
+### Create a Github Actions workflow
```

Comment:
I suggest making the steps in this sub-section a numbered list, as it will make it easier to read. I've added suggestions inline.